api/managementpb/checks.proto

```diff
@@ -28,6 +29,38 @@ message SecurityCheckResult {
   string service_name = 6;
 }
 
+// CheckResultSummary is a summary of check results.
+message CheckResultSummary {
+  string service_name = 1;
+  string service_id = 2;
+  // Number of failed checks for this service that are critical
+  uint32 critical_count = 3;
+  // Number of failed checks for this service that are major
+  uint32 major_count = 4;
+  // Number of failed checks for this service that are trivial
+  uint32 trivial_count = 5;
```

Comment:
I thought we decided to create counter for each possible severity level, no?

Reply:
Yeah,  but that's when frontend is ready to support all the severity levels too.

Reply:
But on the API level we can add all of them now, but fill only these 3.